### PR TITLE
feat: [Tree] virtual tree adds scrollTo method #1024

### PR DESCRIPTION
**Pull request by YyumeiZhang**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- Feat: Tree 增加支持虚拟化场景下的scrollTo方法 #1024 

---

🇺🇸 English
- Fix: Tree adds support for scrollTo method in virtualization scenarios  #1024 

### Checklist
- [x] Test or no need
- [x] Document or no need
- [x] Changelog or no need

### Other
- [ ] Skip Changelog

### Additional information

修改文档见https://bytedance.feishu.cn/docx/doxcn1TsJ5cy7KAlYCdN3PtDWCf
https://user-images.githubusercontent.com/101110131/183796153-e8ed69d0-8e82-48fe-a23f-892c0794b0fc.mp4

### CODE REVIEW

File: content/navigation/tree/index.md [hunk 1]
```diff
@@ -2190,6 +2190,7 @@ import { IconFixedStroked, IconSectionStroked, IconAbsoluteStroked, IconInnerSec
 |名称 | 描述 | 类型
 |----|----|----|
 | search | 手动触发搜索 | (value: string) => void |
+| scrollTo | 在虚拟化 Tree 中，使得指定结点滚动到视图 | （{key: string; align?: 'center' \| 'start' \| 'end' \| 'smart' \| 'auto';}) => void |
```
Comment (pointhalo) [inline content/navigation/tree/index.md:2193]: 如果未expand，直接调用scrollTo，会自动展开对应节点吗

Comment (YyumeiZhang) [reply content/navigation/tree/index.md:2193]: 没有expande的节点无法通过scrollTo实现跳转到该节点。不会对未expand的节点做展开动作

Comment (pointhalo) [reply content/navigation/tree/index.md:2193]: 那可以注明一下。防杠。

File: packages/semi-ui/tree/_story/tree.stories.js [hunk 2]
```diff
@@ -2398,4 +2398,153 @@ export const ValueImpactExpansionWithDynamicTreeData = () => {
...
+    onInputBlur(e) {
+      const { value } = e.target;
+      this.onScroll(value, this.state.align);
+    }
+
+    onSelectChange(value){
+      this.setState({
+        align: value,
+      })
+      this.onScroll(this.state.scrollKey, value)
```
Comment (YannLynn) [inline packages/semi-ui/tree/_story/tree.stories.js:2482]: 后面的value 改成 align 代码可读性更强

**Pull request merged**